src/keystore/InMemoryKeystore.ts

```diff
@@ -23,12 +23,33 @@ import {
   getKeyMaterial,
   topicDataToConversationReference,
 } from './utils'
-import { nsToDate } from '../utils'
+import { nsToDate, buildDirectMessageTopicV2 } from '../utils'
 import InviteStore from './InviteStore'
 import { Persistence } from './persistence'
 import LocalAuthenticator from '../authn/LocalAuthenticator'
+import { hmacSha256Sign } from '../crypto/ecies'
+import crypto from '../crypto/crypto'
 const { ErrorCode } = keystore
 
+// Constant 32 byte salt
+const inviteSalt = new TextEncoder().encode('This is my salt for your invite.')
```

Comment:
We probably want to change this to something that feels more trustworthy. Can be literally any string.

Reply:
`______________xmtp______________`?

Reply:
Long as it adds up to 32 bytes and looks reasonable, I'm down

Reply:
what do you think of this? [fix: update invite salt](https://github.com/xmtp/xmtp-js/pull/326/commits/bbed5fca1fc940614069312aa2ab75c09b4883a8)

Reply:
Works for me